server/game/cards/08.6-SAT/MaceTyrell.js

```diff
@@ -0,0 +1,45 @@
+// https://i62.servimg.com/u/f62/19/86/16/33/28166710.jpg
+
+const DrawCard = require('../../drawcard.js');
+
+class MaceTyrell extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardKneeled: event => event.card === this
+            },
+            handler: () => {
+                this.controller.drawCardsToHand(2);
+                this.game.addMessage('{0} uses {1} to draw 2 cards', this.controller, this);
+
+                this.game.promptForSelect(this.controller, {
+                    numCards: 1,
```

Comment:
The numCards property isn't needed, and I think it causes the prompt to not fire on selection as well (meaning it requires an extra click on the 'Done'-button currently).